offline/p300_paradigm.py

```diff
@@ -13,17 +13,26 @@
 
 from pylsl import local_clock, StreamInfo, StreamOutlet
 
+# Load the configurations of the paradigm
+with open(r'configs/config_long_red-yellow.txt', 'rb') as file:
+    params = pickle.load(file)
+
 # Define custom colors
-GREY = (103, 103, 110)
+GREY = (0, 0, 0)
 WHITE = (255, 255, 255)
-RED = (255, 0, 0)
-YELLOW = (255, 255, 0)
 
-WINDOW_WIDTH = 600 # 600
-WINDOW_HEIGHT = 600 # 600
-# Load the configurations of the paradigm
-with open(r'./config_durations.pkl', 'rb') as file:
-    exp_durations = pickle.load(file)
+TARGET = params['target_color']
+HIGHLIGHT = params['highlight_color']
```

Comment:
Does it make sense to define one variable as another? Isn't it more clear to stick with the initial variable name? Also, I think it would be more clear if you were the name it `TARGET_COLOUR` and `HIGHLIGHT_COLOUR` rather than `TARGET` and `HIGHLIGHT` as then it is ambiguous what is exactly contains (the target itself, the string, the colour, the duration, etc.)